docs/compatibility_matrix.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility Matrix
+
+This document shows the compatibility of Scalar DB with supported databases.
```

Comment:
```suggestion
Scalar DB supports the following databases and the databases that are compatible with those databases.
* aaa
* bbb 
```

Reply:
Thank you for your suggestion.
Suggestion integrated.